powerlevel9k.zsh-theme

```diff
@@ -567,6 +567,61 @@ prompt_context() {
   "$1_prompt_segment" "${0}_${current_state}" "$2" "$DEFAULT_COLOR" "${context_states[$current_state]}" "${content}"
 }
 
+################################################################
+# User: user (who am I)
+# Note that if $DEFAULT_USER is not set, this prompt segment will always print
+set_default POWERLEVEL9K_USER_TEMPLATE "%n"
+prompt_user() {
+  local current_state="DEFAULT"
+  typeset -AH user_state
+  if [[ "$POWERLEVEL9K_ALWAYS_SHOW_CONTEXT" == true ]] || [[ "$USER" != "$DEFAULT_USER" ]] || [[ -n "$SSH_CLIENT" || -n "$SSH_TTY" ]]; then
+    if [[ $(print -P "%#") == '#' ]]; then
+      user_state=(
+        "STATE"               "ROOT"
+        "CONTENT"             "${POWERLEVEL9K_USER_TEMPLATE}"
+        "BACKGROUND_COLOR"    "${DEFAULT_COLOR}"
+        "FOREGROUND_COLOR"    "yellow"
+        "VISUAL_IDENTIFIER"   "ROOT_ICON"
+      )
+    elif [[ "$POWERLEVEL9K_ALWAYS_SHOW_USER" == true ]]; then
```

Comment:
I'm a little confused about the use of `ALWAYS_SHOW_CONTEXT` and `ALWAYS_SHOW_USER`, here. How are they different, and why are both used from this segment? What happens if `ALWAYS_SHOW_CONTEXT` is set to false but `ALWAYS_SHOW_USER` is set to true?

I think it might be more straight-forward to only use the `ALWAYS_SHOW_USER` flag in the the `user` segment, and add it to the README to describe what it does. Thoughts?